src/hooks/users.ts

```diff
@@ -1,63 +1,88 @@
-import { useMemo } from "react";
-import { isLoaded } from "react-redux-firebase";
-
-import { User } from "types/User";
-import { ValidStoreAsKeys } from "types/Firestore";
-
-import { usersSelector, usersByIdSelector } from "utils/selectors";
+import { worldUsersSelector, usersByIdSelector } from "utils/selectors";
 import { WithId } from "utils/id";
 
-import { useSelector } from "hooks/useSelector";
-import { useUserLastSeenThreshold } from "hooks/useUserLastSeenThreshold";
-import { useVenueId } from "hooks/useVenueId";
-import { useFirestoreConnect } from "hooks/useFirestoreConnect";
+import { useSelector } from "./useSelector";
+import { useUserLastSeenThreshold } from "./useUserLastSeenThreshold";
+import { useConnectCurrentVenueNG } from "./useConnectCurrentVenueNG";
+import { useVenueId } from "./useVenueId";
+import { useFirestoreConnect, isLoaded } from "./useFirestoreConnect";
 
-const useConnectVenueUsers = () => {
+import { User } from "types/User";
+import { useMemo } from "react";
```

Comment:
Import ordering: 3rd party library should be up the top, `types/` before `utils/`